sender.go

```diff
@@ -166,7 +167,7 @@ func (s *sender) trySend(ctx context.Context, evt eventer) error {
 
 	recvr := func(err error) {
 		duration := s.recoveryBackoff.Duration()
-		tab.For(ctx).Debug("amqp error, delaying " + string(duration/time.Millisecond) + " millis: " + err.Error())
+		tab.For(ctx).Debug("amqp error, delaying " + strconv.FormatInt(int64(duration/time.Millisecond), 10) + " millis: " + err.Error())
```

Comment:
this is included because govet in 1.15 complains about casting from int64 to string (results in a single character based on the byte value of the int)